hardware/i2c.md

```diff
@@ -34,6 +34,7 @@ Overall, this means 0x1D, 0x0E (from v1.3), 0x1E and 0x19 (for the revision) are
 | accessory name | organisation | i2c address(es) used | 
 |----------------|--------------|-----------------------|
 | eg [banana-bit]()|eg Banana enterprises | eg 0x76, 0x29 |
+| [EDU:BIT](https://www.cytron.io/p-edubit)| Cytron Technologies | 0x08 |
```

Comment:
Thanks @kiate, is this a 7-bit address (excludes the read/write bit)?

Reply:
@carlosperate Yes, that's right.

Reply:
Great, in that case could you provide the addresses in the same format as the micro:bit sensors? `0x__ (0x__/0x__)`

@microbit-mark we should update the example to follow this pattern as well.

Reply:
Done!